src/main/java/pvs/app/service/data/TrelloData.java

```diff
@@ -0,0 +1,60 @@
+package pvs.app.service.data;
+
+import java.util.ArrayList;
+
+public class TrelloData {
+    private final ArrayList<TrelloList> lanes;
+
+    public TrelloData(){
+        this.lanes = new ArrayList<>();
+    }
+
+    public TrelloList createList(String id, String title, String label, int width) {
+        return new TrelloList(id, title, label, width);
+    }
+
+    public void addList(TrelloList trelloList) {
+        this.lanes.add(trelloList);
+    }
+
+    public static class TrelloList {
+        private final String id;
+        private final String title;
+        private final String label;
+        private final Width style;
+        private final ArrayList<Card> cards;
+
+        public TrelloList(String id, String title, String label, int width) {
+            this.id = id;
+            this.title = title;
+            this.label = label;
+            this.style = new Width(width);
+            this.cards = new ArrayList<>();
+        }
+
+        public void addCard(String id, String title, String label, String description) {
+            Card card = new Card(id, title, label, description);
+            this.cards.add(card);
+        }
+    }
+
+    private static class Width {
```

Comment:
Do we have chances to add more styles in the future?
If so, suggest create a interface `Style`, and make `Width` to implement that.

Reply:
I think we don't have a chance to add more style. The json data is fixed format.
It is possible to refactor the class if we don't use `react-trello`.